engine/renderer/public/pipelines/fxaa_pipeline.hpp

```diff
@@ -0,0 +1,55 @@
+﻿#pragma once
+
+#include "frame_graph.hpp"
+#include "gbuffers.hpp"
+
+#include <cstdint>
+#include <memory>
+
+class BloomSettings;
+class GraphicsContext;
+class CameraResource;
+struct GPUImage;
+struct RenderSceneDescription;
+
+class FXAAPipeline final : public FrameGraphRenderPass
+{
+public:
+    FXAAPipeline(const std::shared_ptr<GraphicsContext>& context, const GBuffers& gBuffers, const ResourceHandle<GPUImage>& fxaaTarget, const ResourceHandle<GPUImage>& sourceTarget);
+    ~FXAAPipeline() final;
+
+    void RecordCommands(vk::CommandBuffer commandBuffer, uint32_t currentFrame, const RenderSceneDescription& scene) final;
```

Comment:
nitpick The "final" keyword here is not needed, do specify the functions as override